README.md

```diff
@@ -12,6 +12,14 @@ This sample includes three projects that are meant to help when getting started
 * Upload Documents to an Azure Search index
 * Search & Filter documents within an index
 
+The second sample DotNetHowToIndexers is a simple .NET Core console application that shows how to:

```

Comment:
DotNetHowToIndexer is the **_second_** sample now... so you need to probably update the ordering of all other examples below... or simply move this to the end?